tests/integration/IntegrationSpec.hs

```diff
@@ -31,6 +31,7 @@ import           RIO.Process
                    )
 import qualified RIO.Set as Set
 import qualified RIO.Text as T
+import           RIO.Time ( getCurrentTime, diffUTCTime, formatTime, defaultTimeLocale )
```

Comment:
If you happen to make other revisions (see below), it would be good to list the imports in alphabetical order, for consistency.